src/components/bulk-email-tool/task-alert-modal/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './TaskAlertModal';
```

Comment:
I moved the `TaskAlertModal` to its own folder with its own public API because its now used in more places then just local to the `BulkEmailForm`